pkg/request/pending.go

```diff
@@ -123,7 +173,7 @@ func (ps *PendingStore) checkFirstStrike(now time.Time) {
 	go func() {
 		for _, bucket := range buckets {
 			bucket.requests.Range(func(_, v interface{}) bool {
-				ps.FirstStrikeCallback(v.([]byte))
+				ps.FirstStrikeCallback(v.([]byte), ps.Inspector.RequestID(v.([]byte)))
```

Comment:
why do we need the second parameter? is that only for logging?

Reply:
Yes
Maybe I'll remove it and log here